packages/javascript/bh-shared-ui/src/views/ZoneManagement/Details/DetailsList.tsx

```
@@ -60,6 +62,12 @@ type DetailsListProps = {
 export const DetailsList: FC<DetailsListProps> = ({ title, listQuery, selected, onSelect }) => {
     const [sortOrder, setSortOrder] = useState<SortOrder>('asc');
 
+    const { data } = useGetConfiguration();
+    const tieringConfig = parseTieringConfiguration(data);
+    const multiTierAnalysisEnabled = tieringConfig &&
+        tieringConfig?.value.multi_tier_analysis_enabled;
```

Biome (1.9.4)

[error] Change to an optional chain.

Unsafe fix: Change to an optional chain.


(lint/complexity/useOptionalChain)